Modules/Loadable/Segmentations/EditorEffects/Python/SegmentEditorMarginEffect.py

```diff
@@ -70,17 +66,6 @@ def createCursor(self, widget):
   def setMRMLDefaults(self):
     self.scriptedEffect.setParameterDefault("MarginSizeMm", 3)
 
-  def getKernelSizePixel(self):
```

Comment:
It would be nice to keep (or add?) display of actual margin size in addition to requested margin size.